settings.gradle

```diff
@@ -11,3 +11,5 @@ pluginManagement {
 }
 
 rootProject.name = 'Artemis'
+
+include 'supporting_scripts:analysis-of-endpoint-connections'
```

Comment:
do we need this include here?

Reply:
Yes. It allows us to build and run the subproject in the GitHub action. If we don't include the subproject in the main project, Github _Cannot locate tasks that match ':supporting_scripts:analysis-of-endpoint-connections:run' as project 'supporting_scripts' not found in root project 'Artemis'_.